cfgov/unprocessed/js/modules/Analytics.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+var Analytics = {
+
+  tagManagerIsLoaded: false,
+
+  EVENT_CATERGORY: 'CFGOV Event',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2308/commits/fbcb63f708e0ebc537601faeb0357d7e18e23b44